app/src/test/java/com/concordia/mcga/factories/ShadowMainActivity.java

```diff
@@ -0,0 +1,18 @@
+package com.concordia.mcga.factories;
+
+import android.content.Context;
+
+import com.concordia.mcga.activities.MainActivity;
+
+import org.mockito.Mockito;
+import org.robolectric.annotation.Implementation;
+import org.robolectric.annotation.Implements;
+
+@Implements(MainActivity.class)
+public class ShadowMainActivity {
```

Comment:
The reason this was done is because the Shadow is specific to these tests. It may not be applicable to other Robolectric tests...